web/templates/compose.hbs

```diff
@@ -69,7 +69,7 @@
                                 </button>
                             </div>
                             <div id="compose-direct-recipient" class="pill-container" data-before="{{t 'You and' }}">
-                                <div class="input" contenteditable="true" id="private_message_recipient" data-no-recipients-text="{{t 'Add one or more users' }}" data-some-recipients-text="{{t 'Add another user...' }}"></div>
+                                <div class="input" contenteditable="true" id="private_message_recipient" maxlength="{{ max_topic_length }}" data-no-recipients-text="{{t 'Add one or more users' }}" data-some-recipients-text="{{t 'Add another user...' }}"></div>
```

Comment:
So the `maxlength` attribute is [only valid](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength) on `<input>` and `<textarea>` elements; since all this is really here for is to reach into the DOM from `compose_recipient.js`, why not just access the `max_topic_length` value directly from `compose_recipient.js`?

Reply:
Thank you for the review Karl. I have fixed this issue so the `max_topic_length` is retrieved directly in `compose_recipient.js` with `page_params.max_topic_length`.